src/frontend/apps/lti_site/components/SelectContent/SelectContentTabs/index.spec.tsx

```diff
@@ -235,16 +239,18 @@ describe('SelectContentTabs', () => {
       name: 'Other custom app tab',
     });
     userEvent.click(otherCustomAppTab);
-    screen.getByText('Other select app content');
+    screen.queryByText('Other select app content');
```

Comment:
Why this change ?

Reply:
I think that's because I added a state to manage the tabs. We have to wait for the tab content to be visible.

Reply:
in that case it should be `await screen.findByText('Other select app content');` ?